<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tool_ee</class>
 <widget class="QWidget" name="tool_ee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EE Bin Converter</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Bin File</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>This utility will load and save EE bins in various formats.  The bins loaded/saved/modified here are not automatically loaded into an active flash tool, you have to do that yourself.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Storage Format</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="format_std_btn">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This format is used in TunerPRO (with any sane XDF file), TunerCAT, and probably any other bin editing tool you'll see.  It is T-Side first, E-Side second, and the bin is offset by 8k.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Standard</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="format_lt1edit_btn">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is an E-Side first, T-Side second image only used by LT1edit.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>LT1Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="format_eside_btn_2">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the unscrambled bin (like the ordinary storage format, but with the one ECM side alone.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>E-Side</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="format_tside_btn_2">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the unscrambled bin (like the ordinary storage format, but with the one ECM side alone.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>T-Side</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="format_eside_btn">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the raw EEPROM content..  The format is interpolated as required, as the address lines in the 8051 ECM are scrambled.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Raw E-Side</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="format_tside_btn">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the raw EEPROM content..  The format is interpolated as required, as the address lines in the 8051 ECM are scrambled.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Raw T-Side</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>T-SIDE STATE</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="tside_status"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="eside_status"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>E-SIDE STATE</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Operations</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="load_btn">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load a new bin file in the storage format specified.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Load...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="save_btn">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save the bin file in the format specified.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Save...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clear_btn">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="checksum_btn">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Make the checksum of the loaded bin 'valid' by recalculating it.  You shouldn't do this if the checksum is invalid for an unknown reason, as you may flash garbage onto your ECM.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Repair Checksum</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPlainTextEdit" name="bin_info"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
